internal/util/util.go

```diff
@@ -100,14 +102,14 @@ func InspectPlatformOperators(POList *platformv1alpha1.PlatformOperatorList) err
 func inspectPlatformOperator(po platformv1alpha1.PlatformOperator) error {
 	applied := meta.FindStatusCondition(po.Status.Conditions, platformtypes.TypeApplied)
 	if applied == nil {
-		return buildPOFailureMessage(po.GetName(), platformtypes.ReasonApplyPending)
+		return buildPOMessage(po.GetName(), platformtypes.ReasonApplyPending)
 	}
 	if applied.Status != metav1.ConditionTrue {
-		return buildPOFailureMessage(po.GetName(), applied.Reason)
+		return buildPOMessage(po.GetName(), applied.Reason)
 	}
 	return nil
 }
 
-func buildPOFailureMessage(name, reason string) error {
-	return fmt.Errorf("encountered the failing %s platform operator with reason %q", name, reason)
+func buildPOMessage(name, reason string) error {
+	return fmt.Errorf("%w: %s platform operator has a current reason of %v", ErrorUnreadyPlatformOperator, name, reason)
```

Comment:
This seems a little awkward too. Maybe something like `platform operator is %s, reason` is more legible?

Reply:
Maybe we can agree on `%s platform operator has %v reason`?

Reply:
I'd like @timflannagan to weigh in